io_scene_nif/modules/nif_export/property/object/__init__.py

```diff
@@ -123,10 +128,54 @@ def get_matching_block(self, block_type, **kwargs):
                 setattr(block, param, attribute)
         return block
 
-    # TODO [material][property] Move this to new form property processing
+    def export_properties(self, b_obj, b_mat, n_block):
+        if b_obj and b_mat:
+            # export and add properties to n_block
+            for prop in (self.export_alpha_property(b_mat),
+                         self.export_wireframe_property(b_obj),
+                         self.export_stencil_property(b_mat),
+                         self.export_specular_property(b_mat),
+                         self.material_property.export_material_property(b_mat)
```

Comment:
Is export_vertex_color_property and export_z_buffer_property not here just becuase they aren't support or called from a different context?

Reply:
They need their call and return behavior to be updated, but historically they were only exported for root nodes and only for some versions. Might want to change that. Ie. add a vcol prop if we have vcols on a mesh, and add a zbuff prop if whatever suitable condition is met.